src/pymodaq_plugins_keithley/__init__.py

```diff
@@ -1,8 +1,18 @@
+import os
 from pathlib import Path
-from pymodaq.utils.logger import set_logger  # to be imported by other modules.
+from pymodaq.utils.logger import set_logger
+import pymodaq_plugins_keithley as plugin
+import pymodaq_plugins_keithley.utils as utils
+logger = set_logger('plugin.__init__', add_to_console=False)
 
-from .utils import Config
-config = Config()
+from pymodaq_plugins_keithley.utils import Config_keithley
```

Comment:
do not use a dedicated config file but rather a dedicated entry in the built_in plugin config

Reply:
Done !